### PR TITLE
python3Packages.stevedore: 3.2.2 -> 3.3.0

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 3.3.0

Change log: https://docs.openstack.org/stevedore/latest/user/history.html

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 120274` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>python38Packages.optuna</li>
    <li>python39Packages.optuna</li>
  </ul>
</details>
<details>
  <summary>22 packages built:</summary>
  <ul>
    <li>checkov</li>
    <li>jenkins-job-builder (python38Packages.jenkins-job-builder)</li>
    <li>nox</li>
    <li>papis (python38Packages.papis)</li>
    <li>python38Packages.bandit</li>
    <li>python38Packages.bugwarrior</li>
    <li>python38Packages.cliff</li>
    <li>python38Packages.doc8</li>
    <li>python38Packages.dogpile_cache</li>
    <li>python38Packages.stevedore</li>
    <li>python38Packages.subliminal</li>
    <li>python38Packages.virtualenvwrapper</li>
    <li>python39Packages.bandit</li>
    <li>python39Packages.bugwarrior</li>
    <li>python39Packages.cliff</li>
    <li>python39Packages.doc8</li>
    <li>python39Packages.dogpile_cache</li>
    <li>python39Packages.jenkins-job-builder</li>
    <li>python39Packages.papis</li>
    <li>python39Packages.stevedore</li>
    <li>python39Packages.subliminal</li>
    <li>python39Packages.virtualenvwrapper</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 120274` at 19e052d0 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>python38Packages.optuna</li>
    <li>python39Packages.optuna</li>
  </ul>
</details>
<details>
  <summary>22 packages built successfully:</summary>
  <ul>
    <li>checkov</li>
    <li>jenkins-job-builder (python38Packages.jenkins-job-builder)</li>
    <li>nox</li>
    <li>papis (python38Packages.papis)</li>
    <li>python38Packages.bandit</li>
    <li>python38Packages.bugwarrior</li>
    <li>python38Packages.cliff</li>
    <li>python38Packages.doc8</li>
    <li>python38Packages.dogpile_cache</li>
    <li>python38Packages.stevedore</li>
    <li>python38Packages.subliminal</li>
    <li>python38Packages.virtualenvwrapper</li>
    <li>python39Packages.bandit</li>
    <li>python39Packages.bugwarrior</li>
    <li>python39Packages.cliff</li>
    <li>python39Packages.doc8</li>
    <li>python39Packages.dogpile_cache</li>
    <li>python39Packages.jenkins-job-builder</li>
    <li>python39Packages.papis</li>
    <li>python39Packages.stevedore</li>
    <li>python39Packages.subliminal</li>
    <li>python39Packages.virtualenvwrapper</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>

Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/stevedore/default.nix:32:3:
```
   |
32 |   meta = with lib; {
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/stevedore/default.nix [hunk 1]
```diff
@@ -10,23 +10,28 @@
...
+    importlib-metadata
+  ];
 
   doCheck = false;
   pythonImportsCheck = [ "stevedore" ];
 
   meta = with lib; {
     description = "Manage dynamic plugins for Python applications";
-    homepage = "https://pypi.python.org/pypi/stevedore";
+    homepage = "https://docs.openstack.org/stevedore/";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/stevedore/default.nix:34]: Missing maintainer.

Comment (fabaff) [reply pkgs/development/python-modules/stevedore/default.nix:34]: I've added myself.

**Pull request merged**